pjsip/src/pjsua-lib/pjsua_acc.c

```diff
@@ -1560,14 +1560,13 @@ PJ_DEF(pj_status_t) pjsua_acc_send_request(pjsua_acc_id acc_id,
     PJ_ASSERT_RETURN(dest_uri, PJ_EINVAL);
     PJ_ASSERT_RETURN(method, PJ_EINVAL);
     PJ_UNUSED_ARG(options);
-    PJ_ASSERT_RETURN(msg_data, PJ_EINVAL);
```

Comment:
While here, perhaps also update acc id verification, e.g:
```
PJ_ASSERT_RETURN(pjsua_acc_is_valid(acc_id), PJ_EINVAL);
```
Or simply remove the incomplete acc id check above & add info that the acc verification will be done by `pjsua_acc_create_request()`.

Also, perhaps better to protect the acc verification & usage using PJSUA lock.